src/sprite_battler.cpp

```diff
@@ -264,19 +268,19 @@ bool Sprite_Battler::IsIdling() {
 }
 
 int Sprite_Battler::GetWidth() const {
-	if (animation) {
-		return animation->GetWidth();
-	}
+	if (animation) return 48;
 	return Sprite::GetWidth();
 }
 
 int Sprite_Battler::GetHeight() const {
-	if (animation) {
-		return animation->GetHeight();
-	}
+	if (animation) return 48;
```

Comment:
More on this sprite stuff @rueter37 
First: ``GetWidth/GetHeight`` don't return any meaningful here in general because this is the dimension of the entire animation file, so not useful here.
You forgot to handle the case that it is a large file: https://github.com/EasyRPG/Player/blob/master/src/battle_animation.cpp#L50
Maybe add two new functions ``GetAnimationCellWidth/GetAnimationCellHeight`` that handle this. (yeah they return the same right now, but maybe changes later)

Reply:
Added the functions now.